src/components/layout/ScrollScreen.tsx

```diff
@@ -82,6 +82,7 @@ const ScrollScreen = ({
         onScroll={hasNavigationHeader ? navigationScrollHandler : screenScrollHandler}
         onScrollEndDrag={scrollEndHandler}
         showsHorizontalScrollIndicator={false}
+        keyboardShouldPersistTaps="always"
```

Comment:
Mhh are we sure we want this to be true for all ScrollScreen? 

It means that tapping on a ScrollScreen will never dismiss the keyboard. Is there a specific reason to add this here instead of on a "per screen basis"?

Reply:
This is indeed an issue: try generating a new address and choosing a group :)

Reply:
Ah, I see, it was solving a few issues in the Assets screen. I'll make sure to move it there specifically! Thanks for finding this!

Reply:
Is it going to be fixed in the upcoming PR then?

Reply:
Correct!